libstuff/libstuff.cpp

```diff
@@ -2615,6 +2615,10 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     if ((int64_t)elapsed > warnThreshold) {
         // This code removing authTokens is a quick fix and should be removed once https://github.com/Expensify/Expensify/issues/144185 is done.
         pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("\"authToken\":<REDACTED>", &sqlToLog);
+
+        // We remove anything inside "html" because it's most likely sensitive data that should not be seen in logs
+        pcrecpp::RE("\"html\":\".*\"").GlobalReplace("\"html\":<REDACTED>", &sqlToLog);
```

Comment:
NAB but should we wrap REDACTED in quotes so we don't break the JSON format?  
```suggestion
        pcrecpp::RE("\"html\":\".*\"").GlobalReplace("\"html\":\"<REDACTED>\"", &sqlToLog);
```

Reply:
If we're going to remove everything after "html": then forget about this